services/tenant-ui/frontend/src/components/innkeeper/reservation/ReservationsHistory.vue

```diff
@@ -212,10 +161,7 @@ const formattedReservationHistory = computed(() =>
     state: msg.state,
     reservation_id: msg.reservation_id,
     contact_email: msg.contact_email,
-    contact_name: msg.contact_name,
-    contact_phone: msg.contact_phone,
     tenant_name: msg.contact_name,
```

Comment:
Can do the same fix `tenant_name` instead of contact_name while we're at it on this table too.

Reply:
Done.